src/components/dashboard/Chat/ChatController.js

```diff
@@ -172,19 +173,21 @@ class ChatController extends React.Component {
             {this.state.privateChannels && privateChannels}
             <div id='messageContainer'>
               <ChatMessages
-                path={conversant ? null : this.props.match.url}
                 user={this.props.user}
                 chat={this.props.chat}
                 edit={this.state.edit}
                 setEdit={this.setEdit}
                 saveEdit={this.saveEdit}
                 finishEdit={this.finishEdit}
-                editText={this.state.editText}
+                mentors={this.props.mentors}
                 like={this.props.likeMessage}
+                editText={this.state.editText}
                 reciepient={this.props.conversant}
-                conversantAvatar={this.props.conversantAvatar}
+                deleteMessage={this.deleteMessage}
+                onlineStatus={this.props.onlineStatus}
                 initiatePrivateChat={this.initiatePrivateChat}
-                deleteMessage={this.deleteMessage} />
+                conversantAvatar={this.props.conversantAvatar}
+                path={conversant ? null : this.props.match.url} />
```

Comment:
@bonham000 :point_up: What's up with all the red? I think it's because of [this](https://github.com/bonham000/alumni-network/blob/d774fbaf3c7501cb9b41c1fd9becf9d9b66fe037/src/components/dashboard/Chat/ChatController.js#L141). No error I guess, just the way it's showing it in the JSX code. Maybe escaping the apostrophe would fix this and make the diffs easier to figure out.